modules/client-akka/src/main/scala/be/wegenenverkeer/atomium/client/AbstractAtomiumFeedClientActor.scala

```diff
@@ -0,0 +1,237 @@
+package be.wegenenverkeer.atomium.client
+
+import java.time.LocalDateTime
+
+import akka.actor.{ActorLogging, ActorRef}
+import akka.persistence.{PersistentActor, RecoveryCompleted, SnapshotOffer}
+import be.wegenenverkeer.atomium.client.ImplicitConversions._
+import be.wegenenverkeer.atomium.japi
+import be.wegenenverkeer.atomium.japi.client.FeedEntry
+import be.wegenenverkeer.atomium.client.AbstractAtomiumFeedClientActor.Protocol.HasPosition
+import com.fasterxml.jackson.databind.JsonNode
+import play.api.libs.json._
+import rx.lang.scala.{Observable, Subscriber}
+
+import scala.collection.immutable
+import scala.util.Try
+
+/**
+ * Generic Actor that consumes Atomium feeds (in json format) via the AtomiumClient.
+ * If the feed handling goes wrong, the actor will stop (whatever the failure type is).
+ * It is therefore suggested to run this actor within a BackoffSupervisor or similar construct.
+ *
+ * This actor only sends messages to itself. The feed entries are rapported by the [[publishEvent]] method.
+ *
+ * It's the job of the subclass to decide what to do with these entries.
+ *
+ * @param config feed and actor config
+ * @param eventFormat format for EventBus events
+ * @param feedEntryFormat Reads for reading entries in the feed
+ * @tparam EVENT EVENT type that will be put on the eventbus
+ * @tparam FEEDENTRY FEEDENTRY type in the feed
```

Comment:
FEEDENTRY is a confusing type parameter name here. It corresponds to the [T] parameter in FeedEntry[T] if I undertand correctly